openslides/core/static/js/core/site.js

```diff
@@ -86,6 +86,116 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+//Mark all config strings for translation in Javascript
+//see config_variables.py of each Django app
+.config([
+    'gettext',
+    function (gettext) {
+        //Agenda
+        gettext('Numbering prefix for agenda items');
+        gettext('This prefix will be set if you run the automatic agenda numbering.');
+        gettext('Agenda');
+        gettext('Invalid input.');
+        gettext('Numeral system for agenda items');
+        gettext('Arabic');
+        gettext('Roman');
+        gettext('Begin of event');
+        gettext('Input format: DD.MM.YYYY HH:MM');
+        gettext('Number of last speakers to be shown on the projector');
+        gettext('List of speakers');
+        gettext('Show orange countdown in the last x seconds of speaking time');
+        gettext('Enter duration in seconds. Choose 0 to disable warning color.');
+        gettext('Couple countdown with the list of speakers');
+        gettext('[Begin speech] starts the countdown, [End speech] stops the countdown.');
+        //Assignments
+        gettext('Election method');
+        gettext('Automatic assign of method');
+        gettext('Always one option per candidate');
+        gettext('Always Yes-No-Abstain per candidate');
+        gettext('Elections');
+        gettext('Ballot and ballot papers');
+        gettext('The 100 % base of an election result consists of');
+        gettext('Number of ballot papers (selection)');
+        gettext('Number of all delegates');
+        gettext('Number of all participants');
+        gettext('Use the following custom number');
+        gettext('Custom number of ballot papers');
+        gettext('Publish election result for elected candidates only (projector view)');
+        gettext('Title for PDF document (all elections)');
+        gettext('Preamble text for PDF document (all elections)');
+        //core
+        gettext('Presentation and assembly system');
+        gettext('Event name');
+        gettext('<a href="http://www.openslides.org">OpenSlides</a> is a free web '
```

Comment:
For multiline strings in JavaScript you have to use `'...' + '...' ` or `' ...\ ...'`. Then one of the travis errors should be done.